src/reference/asciidoc/metrics.adoc

```diff
@@ -0,0 +1,269 @@
+[[metrics-management]]
+=== Metrics and Management
+
+==== Configuring Metrics Capture
+
+NOTE: Prior to _version 4.2_ metrics were only available when JMX was enabled.
+See <<jmx>>.
+
+To enable `MessageSource`, `MessageChannel` and `MessageHandler` metrics, add an `<int:management/>` bean to the
+application context, or annotate one of your `@Configuration` classes with `@EnableIntegrationManagement`.
+`MessageSource` s only maintain counts, `MessageChannel` s and `MessageHandler` s maintain duration statistics in
+addition to counts.
+See <<mgmt-channel-features>> and <<mgmt-handler-features>> below.
+
+This causes the automatic registration of the `IntegrationManagementConfigurer` bean in the application context.
+Only one such bean can exist in the context and it must have the bean name `integrationManagementConfigurer`
+if registered manually via a `<bean/>` definition.
+
+In addition to metrics, you can control *debug* logging in the main message flow.
+It has been found that in very high volume applications, even calls to `isDebugEnabled()` can be quite expensive with
+some logging subsystems.
+You can disable all such logging to avoid this overhead; exception logging (debug or otherwise) are not affected
+by this setting.
+
+A number of options are available:
+
+[source, xml]
+----
+<int:management
+    default-logging-enabled="false" <1>
+    default-counts-enabled="false" <2>
+    default-stats-enabled="false" <3>
+    counts-enabled-patterns="foo, !baz, ba*" <4>
+    stats-enabled-patterns="fiz, buz" <5>
+    metrics-factory="myMetricsFactory" /> <6>
+----
+
+[source, java]
+----
+@Configuration
+@EnableIntegration
+@EnableIntegrationManagement(
+		defaultLoggingEnabled = "false", <1>
+		defaultCountsEnabled = "false", <2>
+		defaultStatsEnabled = "false", <3>
+		countsEnabled = { "foo", "${count.patterns}" }, <4>
+		statsEnabled = { "qux", "!*" }, <5>
+		MetricsFactory = "myMetricsFactory") <6>
+public static class ContextConfiguration {
+...
+}
+----
+
+<1> Set to `false` to disable all logging in the main message flow, regardless of the log system category settings.
+Set to 'true' to enable debug logging (if also enabled by the logging subsystem).
+
+<2> Enable or disable count metrics for components not matching one of the patterns in <4>.
+
+<3> Enable or disable statistical metrics for components not matching one of the patterns in <5>.
```

Comment:
I don't know how do we worry now about an accuracy of our docs, but this looks like:
<img width="725" alt="2015-07-08 17 16 46" src="https://cloud.githubusercontent.com/assets/1071590/8582552/841b366a-2595-11e5-81a2-c3e082f5a8b8.png">

Both links in the begging works well, but they are strange a bit :smile: 
And is that good for the item number in the end?